details.php

```diff
@@ -0,0 +1,57 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+    <title>ARTIST WEBSITE</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+<body class="body">
+
+<header class="header">
+            <div class="icons magnify">
+                <a href="index.php">
+                    <div class="icon-link-1"><h1>Home</h1>
+                        <box-icon name='home-alt-2' type='solid' color='#edeade'></box-icon>
+                    </div>
+                </a>
+                <a href="https://www.linkedin.com/feed/?trk=guest_homepage-basic_google-one-tap-submit">
+                    <div class="icon link-1"><i class="fa fa-linkedin-square" aria-hidden="true"></i></div>
+                </a>
+                <a href="">
+                    <div class="icon link-1">
+                        <i class="fa fa-download" aria-hidden="true"></i></div>
+                </a>
+</header>
```

Comment:
Just a note, and you don't have to do this yet: you can create a separate `header.php` file and do `require_once('header.php')` to use it in every php file where it's needed